### ISSUE TITLE
Setting to turn off new indentation behavior (in 1.2.0+) per-snippet

### ISSUE THREAD

**Issue by JordanMandel** [yatki/vscode-surround#72]
Is there a setting to turn off the new indentation behavior (either globally or per-surround snippet)?  I have a snippet I use all the time which is to add a parenthesis block and it's broken now because it surrounds the whole line even though I've highlighted a specific block of text

If I had this code,
```python
if some_object is not None or some_other_condition == 42:
   do_something()
```
and then I highlighted the code "`some_object is not None or some_other_condition == 42`", it became:
```python
if (
  some_object is not None or some_other_condition == 42
):
   do_something()
```

Now, no matter what I highlight, it always becomes this, which is not what I want:
```python
(
    if some_object is not None or some_other_condition == 42
):
   do_something()
```

I probably like this new behavior for *most* snippets, but I would really need to turn it off for others, like the one highlighted above.

_Originally posted by @JordanMandel in https://github.com/yatki/vscode-surround/issues/64#issuecomment-883913191_

**Comment by yatki** [comment]
Hmm.. Fair enough.. I think this is a good idea. I'll see what I can do this week or weekend ☝🏻😁

**Comment by yatki** [comment]
Hey @JordanMandel, after giving it a second thought, I decided the fix the root problem instead of disabling new indentation behaviour. Your problem should be resolved with version v1.2.2. 

The release is available on: 
- [VSCode Marketplace](https://marketplace.visualstudio.com/items?itemName=yatki.vscode-surround)
- [GitHub release](https://github.com/yatki/vscode-surround/releases/tag/v1.2.2)

Don't forget to update your extension and leave a star :) And let me know if the problem still persists.
Cheers🖖🏻,
